packages/do-not-wait-for-empty-event-loop/package.json

```diff
@@ -44,5 +43,9 @@
     "url": "https://github.com/middyjs/middy/issues"
   },
   "homepage": "https://github.com/middyjs/middy#readme",
-  "gitHead": "7a6c0fbb8ab71d6a2171e678697de9f237568431"
+  "gitHead": "7a6c0fbb8ab71d6a2171e678697de9f237568431",
+  "devDependencies": {
+    "@middy/core": "2.0.0-alpha.6",
+    "@types/aws-lambda": "^8.10.3"
```

Comment:
is this needed? it's included in @middy/core

Reply:
It's included there as a dev dependency so we need it here as well I believe

Reply:
oh right, it need to ref the lambda context